csl-data.json

```diff
@@ -1,447 +1,440 @@
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "container-author": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "director": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "editor": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "editorial-director": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "illustrator": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "interviewer": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "original-author": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "recipient": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "reviewed-author": {
+                    "$ref": "#/definitions/nameDefinition"
+                },
+                "translator": {
+                    "$ref": "#/definitions/nameDefinition"
+                }
+            }
+        },
+        "numberFields": {
+            "properties": {
+                "chapter-number": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "collection-number": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "edition": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "issue": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number-of-pages": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "number-of-volumes": {
+                    "$ref": "#/definitions/numberDefinition"
+                },
+                "volume": {
+                    "$ref": "#/definitions/numberDefinition"
+                }
+            }
+        },
+        "stringFields": {
+            "properties": {
+                "abstract": {
+                    "type": "string"
+                },
+                "annote": {
+                    "type": "string"
+                },
+                "archive": {
+                    "type": "string"
+                },
+                "archive_location": {
+                    "type": "string"
+                },
+                "archive-place": {
+                    "type": "string"
+                },
+                "authority": {
+                    "type": "string"
+                },
+                "call-number": {
+                    "type": "string"
+                },
+                "citation-label": {
```

Comment:
This is generated by processors, not set on input.

Reply:
So can this be removed?